src/Microsoft.IdentityModel.Tokens/X509EncryptingCredentials.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public class X509EncryptingCredentials : EncryptingCredentials
     {
+        internal const string _useShortNameForRsaOaepKey = "Switch.Microsoft.IdentityModel.UseShortNameForRsaOaepKey";
```

Comment:
Why are we not supposed to be introducing internals? So far the switches we have for Wilson are set-up this way. Refer to https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/909669f4053f485ad94223fa51e2b8be2f61a76f/src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs#L36

Reply:
Internals are okay, they're needed for tests. But we should be careful using internals between the source libraries.

Reply:
Building on Peter's answer, in the past we have used internals between source libraries but this has been in large part what causes our libraries to break when there are mismatching versions and we are now trying to remove existing internals between libraries and avoid adding more. If this will indeed only ever be used for testing then afaik it is fine but if it is between different libraries even in the same repo my understanding is we are trying to avoid that.

Reply:
@JoshLozensky I believe the intent of the team is to not use InternalsVisibleTo for development purposes but, using "internal" keyword it's fine. In this case we would never want anyone to be able to access the name of the switch and try to do funny stuff with it.